sdk/azidentity/default_azure_credential_test.go

```diff
@@ -277,8 +277,8 @@ func TestDefaultAzureCredential_timeoutWrapper(t *testing.T) {
 	for i := 0; i < 2; i++ {
 		// expecting credentialUnavailableError because delay exceeds the wrapper's timeout
 		_, err = chain.GetToken(context.Background(), testTRO)
-		if _, ok := err.(*credentialUnavailableError); !ok {
-			t.Fatalf("expected credentialUnavailableError, got %T: %v", err, err)
+		if _, ok := err.(credentialUnavailable); !ok {
```

Comment:
Hi @chlowell! Is there a reason to leave the credentialUnavailable type unexported? Trying to implement a credentialErrorReporter wrapper for our custom ChainedTokenCredential but can't due to this line.